hrm/heart_rate_monitor.py

```diff
@@ -183,7 +183,6 @@ def find_heart_rate(self):
         t = times[0]
         last_sample = t[-1]
         # sampling frequency
-        fs = t.size/last_sample
+        fs = round(t.size/last_sample)
```

Comment:
This works... could also consider taking the mean of the `diff` of `t`.  These should yield the same values as what you did, but the latter approach can be more flexible when you need to accommodate issues with uneven `dt` intervals.